drizzlepac/tests/acs/test_acs_kernels.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from stsci.tools import teal
+import drizzlepac
+from drizzlepac import astrodrizzle
+from drizzlepac.helpers.mark import require_bigdata
+from drizzlepac.helpers.io import get_bigdata
+
+from ..helpers import BaseACS, raw_from_asn
+
+def pytest_generate_tests(metafunc):
```

Comment:
Since you are only parametrizing one method below, maybe it is easier to read to use https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions decorator instead? Or do you plan to add more stuff to this file?